html/mycakeapp/webroot/js/schedule.js

```diff
@@ -0,0 +1,132 @@
+$(window).on('load', () => {
+
+  //デフォルトで今日の日付にactiveクラスが設定されている
+  $('.dateList .dateListContent:first-child').addClass('active');
+
+  $('.dateListContent').on('click', (e) => {
+    //日付が黄色になる
+    $('.active').removeClass('active');
+    $(e.currentTarget).addClass('active');
+    //日付の表示が変わる
+    $(e.currentTarget).parent().next('h2').html($(e.currentTarget).text());
+    //表示されている上映スケジュールがフェードアウトする
+    $('#movieList').children().fadeTo(500, 0);
+    //選択された日の上映スケジュールが表示される(非同期通信を使用)
+    $.ajax({
+      type: 'GET',
+      url: 'http://localhost:10380/main/ajax',
+      dataType: 'json',
+      data: { date: $(e.currentTarget).val() },
+    }).done((movies) => {
+      console.log('Ajax通信に成功しました');
+      console.log(movies);
+      let checkCount = 0
+      $.each(movies, (key, movie) => {
+        const week = ['日', '月', '火', '水', '木', '金', '土'];
+        //↓schedule.ctpのid=movieList内の要素を構成
+        if (movie['schedules'].length !== 0) {
+          checkCount += 1;
+          //映画タイトル・上映時間・終了予定の要素及びテキスト作成
+          const title = $('<p>')
+            .attr('class', 'title')
+            .text(movie['name']);
+          const screeningTime = $('<p>')
+            .attr('class', 'screeningTime')
+            .text(`[上映時間：${movie['screening_time']}分]`);
+          const finishDate = $('<p>')
+            .attr('class', 'finishDate')
+            .text(`${new Date(movie['finished_at']).getMonth() + 1}月${new Date(movie['finished_at']).getDate()}日(${week[new Date(movie['finished_at']).getDay()]})終了予定`);
+          //映画タイトル・上映時間・終了予定をmovieDetailListのdiv要素に包括
+          const movieDetailList = $('<div>')
+            .attr('class', 'movieDetailList')
+            .append(title)
+            .append(screeningTime)
+            .append(finishDate);
```

Comment:
こちらも同様関数化した方がいいかと思います。
参考)[残余引数 - JavaScript | MDN](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Functions/rest_parameters)、JavaScript本格入門4.5.2 可変長引数の関数を定義する
```suggestion
          // 例なので改良加えていただけると幸いです。
          const tagAppend=function (tagName,className,...plus) {
            return $(tagName)
              .attr('class', className)
              .append(plus);
          }
          const movieDetailList = tagAppend('<div>', 'movieDetailList', title, screeningTime, finishDate);

```